r2-netty/src/main/java/com/linkedin/r2/netty/client/http/HttpChannelPoolFactory.java

```diff
@@ -96,9 +95,9 @@ public HttpChannelPoolFactory(
     _tcpNoDelay = tcpNoDelay;
     _channelPoolWaiterTimeout = channelPoolWaiterTimeout;
 
-    if (!StringUtils.isEmpty(_udsAddress)) {
+    if (!StringUtils.isEmpty(udsAddress)) {
       _bootstrap = new Bootstrap().
-          group(new EpollEventLoopGroup()).
+          group(eventLoopGroup).
           channel(EpollDomainSocketChannel.class).
```

Comment:
Can this be simplified into a single bootstrap assignment, with a ternary for the socket channel class?